### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_Resources_Keyboard:The tab focus is not visible after deploy a server button.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#10621]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#NonBenchmark;#DesktopApp;#MAS2.4.7;#SQL Azure Data Studio;#FTP;#Keyboard;#STP;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.4.7

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate till Resources section "History tab" using tab key,

**Actual:**
The tab focus after deploy a server button is not visible and when screen reader is on the invisible focus make an announcement of a radio button and on the next tab key the focus is on the history tab under Resources section.

**Expected:**
The tab focus after deploy a server button should be on the history tab and the screen reader should make the announcement of it when in focus under Resources section 

**User Impact:**
Keyboard users will not know where the focus has been moved too. 

**Attachment link for Reference:**
[10621A11y_AzureDataStudioStayHealthy_Welcome_Resources_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/4700711/10621A11y_AzureDataStudioStayHealthy_Welcome_Resources_Keyboard.zip)

**Comment by alanrenmsft** [comment]
@v-bbrady please tag this issue in the pr where you are adding tabindex=-1 for the invisible element

**Comment by fsteffi** [comment]
Version: 1.43.0-insider (user setup)
Commit: b4d5a08199e9e0cb0138ce540c086ac32d072019
Date: 2023-03-07T06:31:08.002Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300

#closed;

[Video 3-07 at 17.19.webm](https://user-images.githubusercontent.com/24580570/223414209-67d2532d-1a0a-4a7a-a81b-a88b04831969.webm)
